output.tf

```diff
@@ -1,5 +1,9 @@
-output "data" {
-  value = {
-    application_dashboard = module.application_dashboard
-  }
+# output "data" {
```

Comment:
please have this commented code removed if no need for it anymore, I see you can also have just set permanent output named "application_dashboard" instead of this test output